card/lib/card/format/nest.rb

```diff
@@ -6,6 +6,13 @@ module Nest
       include Subformat
       include Mode
 
+      # @param cardish card mark
+      # @param options [Hash]
+      # @option options [Symbol] view
+      # @option options [Symbol, Array<Symbol>] :hide hide optional subviews
+      # @option options [Symbol, Array<Symbol>] :show show optional subviews
```

Comment:
hide and show can also take single strings/symbols, no?

(Thanks for adding this!!)